README.md

```diff
@@ -20,6 +20,12 @@ git submodule update --init --recursive
 
 This links the main repo to the [BOPS applicants project](https://github.com/unboxed/bops-applicants/).
 
+If it's your first time setting up the project or you are changing the docker file first run:
+
+```sh
+docker compose build 
```

Comment:
We have centralised a lot of docker commands under a [Makefile](https://github.com/unboxed/bops/blob/main/Makefile)

You can use [make build](https://github.com/unboxed/bops/blob/main/Makefile#L5), which will also update the submodule of BOPS-Applicants

Feel free to include more `make` commands.